packages/web-app-files/src/components/SideBar/SideBar.vue

```diff
@@ -186,6 +197,28 @@ export default {
       ) {
         this.close()
       }
+    },
+
+    openPanel(panel) {
+      this.SET_APP_SIDEBAR_ACTIVE_PANEL(panel)
+
+      if (!this.openedPanels.includes(panel)) {
+        this.openedPanels.push(panel)
+      }
+    },
+
+    closePanel() {
+      const activePanel = cloneStateObject(this.appSidebarActivePanel)
+
+      this.SET_APP_SIDEBAR_ACTIVE_PANEL(null)
+
+      setTimeout(() => {
```

Comment:
Can we use `this.$nextTick` here?

Reply:
I need the specific timeout of 400 ms here.